exercises/bracket-push/example.tt

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'bracket_push'
+
+# Test data version:
+# <%= sha1 %>
+class BracketsTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
+    skip<% end %><% if test_case.long_input? %>
+    str = '<%= test_case.splited_input[0] %>'\
+          '<%= test_case.splited_input[1] %>'<% else %>
+    str = '<%= test_case.input %>'<% end %><% if test_case.expected %>
```

Comment:
Can we work toward something like this:

```ruby
class ClockTest < Minitest::Test<% test_cases.each do |test_case| %>
  def <%= test_case.name %>
    <%= test_case.skipped %>
    <%= test_case.test_body %>
  end
<% end %>
```

works with this method in `clock_generator.rb`

```ruby
  def skipped
    index.zero? ? '# skip' : 'skip'
  end
```

That template erb file (.tt) is working with a JSON that has multiple test categories.  But the complexity has been removed from the template.